docs/DeveloperGuide.md

```diff
@@ -1060,6 +1058,31 @@ testers are expected to do more *exploratory* testing.
     (where incomplete details are given for the applicant being added).
     Expected: Error messages displaying the cause of error is shown in the result display.  
 
+### Editing an applicant
+
+1. Editing an applicant while all applicants are being shown
+
+  1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
+
+  1. Test case: `edit 1 n/Alexander p/87654321 e/alexander@gmail.com`<br>
```

Comment:
I think it might be good if you specifically included the Alexander and Alice in the Prerequisite.

Reply:
Hm I think its okay to not include them since regardless of who is in the list, I want to edit their names to become Alexander and Alice, but I will add a prerequisite to say that there shouldn't be anyone with the email and phone number that I want to edit in!

Reply:
Oh I meant for Alice only, since you doing the error issue with the Alice one but no context for why it is duplicate so they cannot replicate your manual testing

Reply:
Oh hmm I meant for the manual testing instructions to be carried out sequentially so Alice one use the same phone number as Alexander from step 1 that's why duplicate but maybe it's not clear. I'll try to make it slightly clearer